content/applications/general/search.rst

```diff
@@ -0,0 +1,101 @@
+=========================
+Search and filter records
+=========================
+
+Odoo uses filters to include only the most relevant records depending on the purpose of the view you
+are on. However, you can edit the default filter or search for specific values.
+
+.. _search/preconfigured-filters:
+
+Preconfigured filters
+=====================
+
+You can modify the default selection of records by clicking :guilabel:`Filters` and selecting one or
+several **preconfigured filters**.
+
+.. example::
+   On the Sales Analysis report, only records at the sales order stage are selected by default.
+   However, you could *also* include records at the quotation stage by selecting
+   :guilabel:`Quotations`. Furthermore, you could *only* include records from a specific year, for
+   example *2022*, by selecting :menuselection:`Order Date --> 2022`.
+
+   .. image:: search/preconfigured-filters.png
+      :align: center
+      :alt: Using preconfigured filters on the Sales Analysis report
+
+.. note::
+   If you select several preconfigured filters that are grouped (i.e., that are *not* separated by
+   a divider, a thin horizontal line), the records can match *any* condition to be included.
+   However, if you select filters from different groups, the records have to match *all* conditions
+   to be included.
+
+.. _search/custom-filters:
+
+Custom filters
+==============
+
+You can create custom filters using most fields present on the model by clicking
+:menuselection:`Filters --> Add Custom Filter`, selecting a field, an operator, a value, and
+clicking :guilabel:`Apply`.
+
+.. example::
+   You could *only* include records from a single salesperson on the Sales Analysis report, for
+   example *Mitchell Admin*, by selecting :guilabel:`Salesperson` as the field, :guilabel:`is equal
+   to` as the operator, and typing `Mitchell Admin` as the value.
+
+   .. image:: search/custom-filter.png
+      :align: center
+      :alt: Using a custom filter on the Sales Analysis report
+
+.. note::
+   If the records should *only* match one of several conditions, click :guilabel:`Add a condition`
```

Comment:
I've noted an inconsistency between "click" and "click on", I don't mind if it differs from one page to another, but I feel like it's weird to switch on the same page